packages/grafana-data/src/types/datasource.ts

```diff
@@ -358,6 +358,7 @@ export interface ExploreQueryFieldProps<
   history: any[];
   onBlur?: () => void;
   absoluteRange?: AbsoluteTimeRange;
+  exploreMode?: 'Logs' | 'Metrics';
```

Comment:
We have existing [ExploreMode enum](https://github.com/grafana/grafana/blob/c4c031ef4368a84fb35412d8729e78db4ede23cd/public/app/types/explore.ts#L21) that we should use here as a type.